docs/basic_usage.rst

```diff
@@ -133,3 +133,39 @@ profiler in your child.
 
 The global profiler instrumented by `ddtrace-run` and `ddtrace.profiling.auto`
 can be started by calling `ddtrace.profiling.auto.start_profiler`.
+
+Asyncio Support
+---------------
+
+The profiler supports the ``asyncio`` library and retrieves the
+``asyncio.Task`` names to tag along the profiled data.
+
+For this to work, the profiler `replaces the default event loop policy
+<https://docs.python.org/3/library/asyncio-policy.html#asyncio-policies>`_ with
+a custom policy that tracks threads to loop mapping.
+
+The custom asyncio loop policy is installed by default at profiler startup. You
+can disable this behavior by using the ``asyncio_loop_policy`` parameter and
+passing it ``None``::
```

Comment:
Should we make this configurable via env variable for those who use `ddtrace-run` (or relies on `.auto`) and don't want to make code changes?